pkg/agent/baker.go

```diff
@@ -216,9 +210,6 @@ func getContainerServiceFuncMap(config *NodeBootstrappingConfiguration) template
 		"IsKubernetesVersionGe": func(version string) bool {
```

Comment:
this is util method, I prefer we keeping it so we can use it freely

Reply:
This comment is on this line:
"IsKubernetesVersionGe": func(version string) bool {

but I didn't remove IsKubernetesVersionGe. Do you meant to comment it on a different function that I removed?